omod-common/src/main/java/org/openmrs/module/webservices/rest/web/RestUtil.java

```diff
@@ -831,13 +831,18 @@ public static SimpleObject wrapErrorResponse(Exception ex, String reason) {
 		StackTraceElement[] steElements = ex.getStackTrace();
 		if (steElements.length > 0) {
 			StackTraceElement ste = ex.getStackTrace()[0];
+			String stackTraceDetailsenabled_gp = Context.getAdministrationService()
+					.getGlobalProperty(RestConstants.ENABLE_STACK_TRACE_DETAILS_GLOBAL_PROPERTY_NAME);
```

Comment:
If this property is NOT defined, what happen? Do you get NULL? If so, you might get a NullPointerException. I think there is an alternative method called getGlobalPropertyValue( name, "defaultValue") which handles the NPE scenario. Please check.

Reply:
@gsluthra yes modified to have default as false .Now if we give some value to it it would consider that else it would always have a default as false to avoid NPE.Please verify